content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -221,71 +221,113 @@ sysctl --system
 {{< tabs name="tab-cri-cri-o-installation" >}}
 {{% tab name="Debian" %}}
 
-```shell
-# Debian Unstable/Sid
-echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Debian_Unstable/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
-wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/Debian_Unstable/Release.key -O- | sudo apt-key add -
-```
+To install CRI-O on the following operating systems, set the environment variable $OS as the appropriate field in the following table:
```

Comment:
Use code format for [inline code](https://kubernetes.io/docs/contribute/style/style-guide/#use-code-style-for-inline-code-and-commands).
```suggestion
To install CRI-O on the following operating systems, set the environment variable $OS to the appropriate field in the following table:
```